lgc/builder/BuilderImpl.cpp

```diff
@@ -35,9 +35,11 @@
 #include "lgc/state/TargetInfo.h"
 #include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/IntrinsicsAMDGPU.h"
+#include "llvm/IR/PatternMatch.h"
```

Comment:
What is this being used for?

Reply:
Ah, I see. Can you use the namespace directly into the method `emitWaterfallBeginForScalarizedLoops`?